core/assets/bundles/bundle_nl.properties

```diff
@@ -928,12 +983,13 @@ block.crawler-factory.name = Crawler Mech Factory
 block.titan-factory.name = Titan Mech Factory
 block.fortress-factory.name = Fortress Mech Factory
 block.revenant-factory.name = Revenant Fighter Factory
-block.repair-point.name = Repair Point
+block.repair-point.name = APK Punt
```

Comment:
What does `APK` mean?

Reply:
Oh that was kind of a dutch inside joke, its where cars go get repaired 😛